src/features/people/peopleSlice.js

```diff
@@ -16,10 +16,11 @@ const peopleSlice = createSlice({
         setLoadingToFail: state => {
             state.loadingStatus = "fail";
         },
+        fetchPeople: () => { },
     },
 });
 
-export const { setPeople, setLoadingToSucces, setLoadingToFail } = peopleSlice.actions;
+export const { setPeople, setLoadingToSucces, setLoadingToFail, fetchPeople } = peopleSlice.actions;
 export const selectPeopleState = state => state.people;
 export const selectPeople = state => selectPeopleState(state).people; 
 export default peopleSlice.reducer;
```

Comment:
I can't see changes from latest merge
so did I miss more changes and now we don't need that? 
``export const selectPageStatus = state => selectPeopleState(state).loadingStatus;``
I add it' to cover page status..